nix/haskell.nix

```diff
@@ -38,6 +38,22 @@ let
   pkgSet = haskell.mkStackPkgSet {
     inherit stack-pkgs;
     modules = [
+      # Allow reinstallation of Win32
```

Comment:
This does not adequately explain it for me ... why is this needed, what does it do? @cleverca22

Reply:
https://github.com/input-output-hk/ouroboros-network/blob/master/nix/haskell.nix#L24-L39
it was copied directly from the same file in `ouroboros-network`
all of the packages in that list, are baked into the ghc, and cant be updated, causing `haskell.nix` to silently ignore the version you specify in `stack.yaml`

the default list includes `Win32`, and this will overwrite that default list

Reply:
Thanks, I have added more detail to the comment.